arlo_server/ballots.py

```diff
@@ -103,3 +106,54 @@ def get_retrieval_list(election: Election, jurisdiction: Jurisdiction, round_id:
         retrieval_list_csv,
         filename=f"ballot-retrieval-{election_timestamp_name(election)}",
     )
+
+
+def serialize_ballot_draw(ballot_draw: SampledBallotDraw) -> JSONDict:
+    # TODO separate status for ballots that were skipped by the audit board
+    ballot = ballot_draw.sampled_ballot
+    audit_board = ballot.audit_board
+    batch = ballot_draw.batch
+    return {
+        "ticketNumber": ballot_draw.ticket_number,
+        "status": "AUDITED" if ballot.vote is not None else None,
+        "vote": ballot.vote,
+        "comment": ballot.comment,
+        "position": ballot.ballot_position,
+        "batch": {"id": batch.id, "name": batch.name, "tabulator": batch.tabulator,},
+        "auditBoard": {"id": audit_board.id, "name": audit_board.name,},
+    }
+
+
+@app.route(
+    "/election/<election_id>/jurisdiction/<jurisdiction_id>/round/<round_id>/ballot-draws",
+    methods=["GET"],
+)
+@with_jurisdiction_access
+def list_ballot_draws_for_jurisdiction(
+    election: Election,  # pylint: disable=unused-argument
+    jurisdiction: Jurisdiction,
+    round_id: str,
+):
+    Round.query.get_or_404(round_id)
+    ballot_draws = (
+        SampledBallotDraw.query.filter_by(round_id=round_id)
+        .join(Batch)
+        .filter_by(jurisdiction_id=jurisdiction.id)
+        .join(SampledBallot)
+        .join(AuditBoard)
+        .order_by(
+            AuditBoard.name,
+            Batch.name,
+            SampledBallotDraw.ballot_position,
+            SampledBallotDraw.ticket_number,
+        )
+        .options(
+            joinedload(SampledBallotDraw.batch),
+            joinedload(SampledBallotDraw.sampled_ballot).joinedload(
+                SampledBallot.audit_board
+            ),
+        )
```

Comment:
@benadida did you know about this feature of sqlalchemy? It's pretty nice!

Reply:
Not this particular formulation, I think I've done it another way?? But in any case yes great :)